src/topology/continuous_function/polynomial.lean

```diff
@@ -0,0 +1,36 @@
+/-
+Copyright (c) 2021 Scott Morrison. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Scott Morrison
+-/
+import topology.algebra.polynomial
+import topology.continuous_function.basic
+
+/-!
+# Constructions relating polynomial functions and continuous functions.
+
+This file is just a stub at the moment, but will grow with subsequent PRs
+giving abstract statements of the Weierstrass approximation theorem,
+and the Stone-Weierstrass theorem.
+-/
+
+namespace polynomial
+
+variables {R : Type*} [semiring R] [topological_space R] [topological_semiring R]
+
+/--
+Every polynomial with coefficients in a topological semiring gives a (bundled) continuous function.
+-/
+def as_continuous_map (p : polynomial R) : C(R, R) :=
```

Comment:
Wouldn't `to_continuous_map` be a more standard formulation in mathlib? Same for the next lemma.

Reply:
Yes!